actions/get_host.yaml

```diff
@@ -1,11 +1,12 @@
 ---
-description: Get Icinga2 host object
+name: get_host
+runner_type: "python-script"
+description: Get all Icinga2 host objects, or individual host
 enabled: true
 entry_point: get_host.py
 parameters:
-  hosts:
-    type: "array"
+  host:
+    type: "string"
```

Comment:
Should add description attribute for this parameter and make it explicit it returns all host objects if this parameter is not provided.